rt_utils/rtstruct.py

```diff
@@ -34,7 +42,16 @@ def add_roi(self, mask: np.ndarray, color: Union[str, List[int]] = None, name: s
         # TODO test if name already exists
         self.validate_mask(mask)
         roi_number = len(self.ds.StructureSetROISequence) + 1
-        roi_data = ROIData(mask, color, roi_number, name, self.frame_of_reference_uid, description, use_pin_hole)
+        roi_data = ROIData(
+            mask,
+            color,
+            roi_number,
+            name,
+            self.frame_of_reference_uid,
+            description,
+            use_pin_hole,
+            approximate_contours
+            )
```

Comment:
Might be best to uses kwargs here so the arguments never end up setting the wrong variable if we change the spec